File: program/find-surface-area-of-a-cylinder/findSurfaceAreaOfACylinder.scala
import scala.io.StdIn._;

object surfaceAreaOfCylinder {
    def main(args:Array[String]):Unit={
        println("Enter the radius : ");
        var r = readLine().toInt;
        println("Enter the height : ");
        var h = readLine().toInt;
        var result = 2 * 3.14 * r * (r+h);
        println("Surface area of Cylinder : "+result);
    }
}

File: program/program/find-surface-area-of-a-cylinder/find_surface_area_of_a_cylinder.dart
import 'dart:io';
void main(){
  print("Enter the Radius :");
  var radius=int.parse(stdin.readLineSync());
  print("Enter the Height :");
  var height=int.parse(stdin.readLineSync());
  var area = 2 * 3.14 * radius * (radius + height);
  print('Area of Cylinder: $area');
}
